ably-ios/ARTDefault.h

```diff
@@ -15,6 +15,11 @@
 + (NSString*)realtimeHost;
 + (int)TLSPort;
 
+/**
+ Default in minutes of requested time to live for the token.
+ */
++ (double)ttl;
```

Comment:
I decided to express it in minutes. I could use `NSTimeInterval`.
This is maybe confusion https://github.com/ably/ably-ios/blob/b4bd0617bb96265e4e9f847bc397e383e33bbba8/ably-ios/ARTAuthTokenParams.m#L23.
What do you suggest?